src/app/content/hooks/locationChange/index.ts

```diff
@@ -2,10 +2,12 @@ import { RouteHookBody } from '../../../navigation/types';
 import { content } from '../../routes';
 import { syncSearch } from '../../search/hooks';
 import resolveContent from './resolveContent';
+import trackPageView from './trackPageView';
 
 const hookBody: RouteHookBody<typeof content> = (services) => async(action) => {
   await resolveContent(services, action.match);
   await syncSearch(services)(action);
+  await trackPageView(action.match);
```

Comment:
Note sure if this is the right place for this.  And pretty sure that `await` isn't needed/desired?

Reply:
i'd make a new hook in `src/app/hooks/trackPageView.ts` and put the call there to match any locationChange action